src/app/util/types_stub.h

```diff
@@ -1928,4 +1928,10 @@ typedef struct
 #define MILLISECOND_TICKS_PER_SECOND 1000
 #define MILLISECOND_TICKS_PER_DECISECOND (MILLISECOND_TICKS_PER_SECOND / 10)
 
+#define emberEventControlSetDelayMS(control, delay) (void) 0
```

Comment:
Work to fix these tracked here
https://github.com/project-chip/connectedhomeip/issues/2431